src/Build/BackEnd/Components/Logging/LoggingServiceLogMethods.cs

```diff
@@ -714,9 +717,10 @@ public void LogTaskStarted(BuildEventContext taskBuildEventContext, string taskN
         /// <param name="projectFileOfTaskNode">Project file which contains the task</param>
         /// <param name="line">The line number in the file where the task invocation is located.</param>
         /// <param name="column">The column number in the file where the task invocation is located.</param>
+        /// <param name="taskAssemblyName">An assembly's unique identity where the task is implemented</param>
```

Comment:
```suggestion
        /// <param name="taskAssemblyName">The unique identity of the assembly containing the implementation of the task.</param>
```

Reply:
I used some existing doc from `AssemblyName` related code for this, but I had to change it anyway as we will log the assembly location. I used your proposal as the template. Thanks.